docs/AMF/using-amf/validation.mdx

```diff
@@ -0,0 +1,53 @@
+---
+id: amf_validation
+title: Validation
+---
+
+To validate a model means checking if it complies with the constraints defined for it. In case it doesn't comply,
+pinpoint where and how the model is invalid, and what severity this issue has.
+
+Once the model has been parsed and resolved, AMF can apply two kinds of validations:
+- **WebApi validations**
```

Comment:
in the particular case of WebApi, we're actually referring to the [WebApi](https://github.com/aml-org/amf/blob/develop/documentation/model.md#webapi) model and it's written like 'WebApi' instead of 'Web API'. But in this case it's talking about more general Web API validations, not necessarily linked to our WebApi model.

I've made the change, but just wanted to make that distinction